src/containers/LargeScaleStructureContainer.jsx

```diff
@@ -34,133 +42,124 @@ class LargeScaleStructureContainer extends React.PureComponent {
 
     API.get(source).then(response => {
       const { data } = response;
-      const { galaxies } = data || {};
-      const [min, max] = extentFromSet(galaxies, 'redshift');
-      const formattedMin = formatValue(min, 2);
+      const { intervals, objects: galaxies } = data || {};
 
       this.setState(prevState => ({
         ...prevState,
-        data: galaxies,
-        sliderVal: formattedMin,
-        min: formattedMin,
-        max: formatValue(max, 2),
-        formattedData: arrayifyData(galaxies),
+        data: colorize(galaxies),
+        intervals,
+        activeInterval: 0,
+        sliderVal: intervals[0][0],
+        min: intervals[0][0],
+        max: intervals[0][1],
```

Comment:
I think I was holding on to these just in case I was still going to use the "redshift slider". This was early on in the development stage. Now that we're at a good place here, I'm going to discard them.